ansible_collections/arista/avd/roles/eos_validate_state/templates/generate_anta_report_results.j2

```diff
@@ -0,0 +1,32 @@
+{# j2lint: disable=V1 #}
```

Comment:
Why?

Reply:
addressed in https://github.com/aristanetworks/ansible-avd/commit/de04df313f6991308683742ffa0528b81d5d8ec9